pom.xml

```diff
@@ -39,8 +39,8 @@
     </modules>
 
 	<properties>
-		<openMRSVersion>1.9.9</openMRSVersion>
-		<openMRSMinorVersion>1.9</openMRSMinorVersion>
+		<openMRSVersion>1.10.5</openMRSVersion>
```

Comment:
You might want to use the latest release in the 1.10.x line which is 1.10.6

Reply:
Thanks for pointing this out @wluyima. The version has been updated.